.gitignore

```diff
@@ -33,3 +33,7 @@ hs_err_pid*
 # .classpath and .project should be generated automatically
 .classpath
 .project
+
+# added IntelliJ rules
+*.iml
+.idea
```

Comment:
This commit

> added intellij rules to .gitignore

is not referenced in the pull request description. Please try to make changes on-point and describe what you've done in your pull request.

Reply:
Sorry about that, haven't used github for a long time...